theseus_gui/src/components/ui/Navigation.vue

```diff
@@ -0,0 +1,148 @@
+<script setup>
+import { SearchIcon, BookIcon, ClientIcon, PlusIcon, SettingsIcon, Button } from 'omorphia'
+import { useInstances } from '@/store/state'
+import { RouterLink } from 'vue-router'
+import UserSection from '@/components/ui/UserSection.vue'
+import Instance from '@/components/ui/Instance.vue'
+const instances = useInstances()
+
+instances.fetchInstances()
+</script>
+
+<template>
+  <div class="nav-container">
+    <UserSection />
+    <div class="pages-list">
+      <RouterLink to="/" class="btn"> <ClientIcon />Home</RouterLink>
+      <RouterLink to="/browse" class="btn"> <SearchIcon />Browse</RouterLink>
+      <RouterLink to="/library" class="btn"> <BookIcon />Library</RouterLink>
+    </div>
+    <div class="instance-list">
+      <p>Instances</p>
```

Comment:
This text should be a little smaller (or the instance text should be bigger) to "stand out" from the individual instance cards. It also could be a more gray-ish color.